### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by Stannnnn** [Staartvin/Statz#24]
I am using Statz for an Achievements plugin. At the moment players have to type a command to manually let my plugin check if they completed certain Achievements. Now isn't this really the most efficient way of doing it because it's manual. Could you create an event that gets thrown whenever you update a record to the database of Statz? So I could use this event to make it automatic. Now I get that calling an event every time you update a record could be much so maybe call the event every x updates? Anyway I'd like it if you would add this.

**Comment by Staartvin** [comment]
What kind of data do you want to get from Statz? I can fire events when updates are sent, but to design the event I need to know what kind of data you want to retrieve from this event. Think about data type, what kind of data, relative or absolute values. 

A tip for automation of your plugin: 
You can also retrieve the data of players every x seconds, so it doesn't wait for an update, but checks every once in a while to see whether meets the achievement requirements. 

For more information on Statz, you can also contact me via Skype (Staartvin)!

**Comment by Stannnnn** [comment]
The achievements are configurable and I added full support for every Statz type. Now if I manually do this every x seconds I need to use your API method really often. Which would be much more intensive then just using events and updating whatever is updated at that moment.

And for the type of event. The more abstract the better. Your API has 1 method to get values so use that same format for launching events?

I added you on Skype but you haven't accepted it yet.
